.github/workflows/ci-test-xtgeo-cibuildwheel.yml

```diff
@@ -3,64 +3,41 @@ name: cibuildwheel
 on:
   pull_request:
       branches: [master, "*postfix"]
+  release:
+    types: [created]
 
-# Verify that wheel build jobs succeed
 jobs:
-  build_linux_cibuildwheel:
+  build_and_upload_with_cibw:
     name: CIBW python ${{ matrix.cibw_python }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-
-    env:
-      CIBW_BEFORE_TEST: >
-        pushd {project} &&
-        pip install -r requirements/requirements.txt &&
-        pip install -r requirements/requirements_test.txt &&
-        git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-      CIBW_TEST_COMMAND: >
-        pushd {project} &&
-        pytest tests/test_common --disable-warnings -x --hypothesis-profile ci --generate-plots
-      CIBW_BUILD: ${{ matrix.cibw_python }}-manylinux_x86_64
-      CIBW_MANYLINUX_X86_64_IMAGE: manylinux2010
-
     strategy:
-      matrix:
-        os: [ubuntu-latest]
-        cibw_python: [cp36, cp39]
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v2
-
-      - name: Install cibuildwheel
-        run: >
-          python -m pip install pip -U &&
-          python -m pip install wheel &&
-          python -m pip install cibuildwheel
-
-      - name: Build wheels and test
-        run: |
-          python -m cibuildwheel --output-dir wheelhouse
-
-  build_macos_cibuildwheel:
-    name: CIBW python ${{ matrix.cibw_python }} on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+     matrix:
+       include:
+           - os: ubuntu-latest
+             cibw_image: manylinux_x86_64
+             cibw_python: cp36
+           - os: ubuntu-latest
+             cibw_image: manylinux_x86_64
+             cibw_python: cp39
+           - os: windows-latest
+             cibw_image: win_amd64
+             cibw_python: cp39
+           - os: macos-latest
+             cibw_image: macosx_x86_64
```

Comment:
Seems that deploy is limited to cp36 and cp39?

Reply:
My mistake, I just kept what was tested wit cibw before. I'll update it to all previously used

Reply:
The former workflow was to limit CIBW tests to 3.6 and 3.9 ("python version endpoints") since all py versions are covered in the ordinary (non-CIBW) tests. Then, while doing deploy with version tagging, CIWB expands to all supported Python versions. Not needed to be like however.

Reply:
Yes, I thought that might be the idea. I reduced it to just the one job for simplicity, and it doesn't really slow us down as the time you have to wait before merging is the slowest running job.

Reply:
What is CIBW?
According to google it's `Cook Inlet Beluga Whales`.

Reply:
Its cibuildwheel.

Reply:
It builds wheel packages for PYPI based on approved libs (i.e. using older CENTOS images for Linux)